docs/notebooks/quickstart.ipynb

```diff
@@ -598,19 +600,26 @@
          ]
       },
       {
+         "attachments": {},
          "cell_type": "markdown",
          "id": "ac3bfe1a",
          "metadata": {},
          "source": [
             "**Credo AI Governance Platform**\n",
             "\n",
-            "For information on how to interact with the plaform, please look into: [Connecting with Governance App](https://credoai-lens.readthedocs.io/en/stable/notebooks/governance_integration.html) tutorial for directions.\n"
+            "For information on how to interact with the plaform, please look into: [Connecting with Governance App](https://credoai-lens.readthedocs.io/en/stable/notebooks/platform_integration.html) tutorial for directions.\n"
          ]
+      },
+      {
+         "cell_type": "markdown",
+         "id": "133c3491",
+         "metadata": {},
+         "source": []
       }
    ],
    "metadata": {
       "kernelspec": {
-         "display_name": "Python 3.9.13 ('.env2': venv)",
+         "display_name": "test_connect_branch",
```

Comment:
Is the `display_name` change intended here?

Reply:
No, but there is no goal for the display name. It's an arbitrary consequence of the last time the notebook was run and saved by someone's commit.